trojsten/results/tests.py

```diff
@@ -119,6 +119,29 @@ def test_invalid_user(self):
         response = self.client.get("%s?show_staff=True" % self.url)
         self.assertContains(response, self.user.get_full_name())
 
+    def test_dont_show_old_result(self):
+        bad_time = timezone.now() + timezone.timedelta(days=-200)
+        competition1 = Competition.objects.create(name='oldCompetition')
+        competition1.sites.add(Site.objects.get(pk=settings.SITE_ID))
+        semester1 = Semester.objects.create(
+            number=1, name='Test semester', competition=competition1, year=1
+        )
+        Round.objects.create(number=1, semester=semester1, solutions_visible=True, visible=True, end_time=bad_time)
+
+        good_time = timezone.now() + timezone.timedelta(days=-47)
+        competition2 = Competition.objects.create(name='newCompetition')
+        competition2.sites.add(Site.objects.get(pk=settings.SITE_ID))
+        semester2 = Semester.objects.create(
+            number=1, name='Test semester', competition=competition2, year=1
+        )
+        Round.objects.create(number=2, semester=semester2, solutions_visible=True, visible=True, end_time=good_time)
+
+        response = self.client.get(self.url)
+
+        names = list(map(lambda x: x.scoreboard.round.semester.competition.name, response.context['scoreboards']))
+        self.assertTrue('newCompetition' in names)
```

Comment:
Existuje `assertIn` a `assertNotIn`
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn

Vyhodu to ma ze ti povie co je zle, ked to neni splnene.

Reply:
mam to esce zmenit ?

Reply:
ano